mkdocs/commands/build.py

```diff
@@ -143,6 +146,7 @@ def get_page_context(page, content, toc, meta, config):
         'meta': meta,
 
         'canonical_url': page.canonical_url,
+        'edit_url': page.edit_url,
```

Comment:
This is deprecated and will be removed in the next release, so lets not even add it here. As the `edit_url` is page specific, then it should only ever be accessible from the `page` object: `page.edit_url`.